scripts/docker-build.sh

```diff
@@ -5,39 +5,101 @@
 #
 SCRIPT_DIR=$(dirname $0)
 BASE_DIR=${PWD}/${SCRIPT_DIR}/..
-PG_VERSION=${PG_VERSION:-10.0}
+PG_VERSION=${PG_VERSION:-9.6.5}
 PG_IMAGE_TAG=${PG_IMAGE_TAG:-${PG_VERSION}-alpine}
 BUILD_CONTAINER_NAME=${BUILD_CONTAINER_NAME:-pgbuild}
 BUILD_IMAGE_NAME=${BUILD_IMAGE_NAME:-$USER/pgbuild}
 IMAGE_NAME=${IMAGE_NAME:-$USER/timescaledb}
 GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD ${BASE_DIR} | awk '{print $1; exit}' | sed -e "s|/|_|g")
-TAG_NAME=${TAG_NAME:-$GIT_BRANCH}
+GIT_TAG=$(git -C ${BASE_DIR} rev-parse --short --verify HEAD)
+GIT_ID=$(git -C ${BASE_DIR} describe --dirty | sed -e "s|/|_|g")
+TAG_NAME=${TAG_NAME:-$GIT_ID}
+BUILD_TYPE=${BUILD_TYPE:-Debug}
+USE_OPENSSL=${USE_OPENSSL:-True}
```

Comment:
minor nit: every other instance of 'true' in these scripts is lowercased, is this uppercase just to make it easier to spot in the output? (since it's not used in a check anywhere)